netlify.toml

```diff
@@ -3,6 +3,7 @@
   # HTML files and assets generated by the build. If a base directory has
   # been specified, include it in the publish directory path.
   publish = "docs/export/"
+  base = "/docs"
```

Comment:
https://docs.netlify.com/configure-builds/overview/#definitions

Gotta be `/` in our case

We could move this file inside `docs` though apparently

Reply:
It seems to work with changing the `base` as well. 🤔

Reply:
fair, it changes where the scripts are run, and `pnpm i` will actually run on the parent if called from a workspace

Reply:
With this setup, wouldn't that mean `pnpm-lock.yaml` doesn't invalidate their cache? `@mui/monorepo` is defined as a top-level dependency, updating it (or any other top-level dependency) wouldn't invalidate cache then?

Reply:
Thanks for all your points, I've read more of the docs and found their monorepo suggestions.
Indeed, using `base` is not in the suggestions scope. 🙈